### PR TITLE
Corrections in threejs-fundamentals.md

### PR DESCRIPTION
**Pull request by a-dpq**
I made some minor corrections to `threejs-fundamentals.md`. On the Three.js website, the name of the library is usually capitalized, so I've used this style throughout.

### CODE REVIEW

**Comment by a-dpq** [comment]
I'd like to go through more lessons in the course and correct them as I do, but only if there's real interest in polishing the text. There are currently a lot of minor mistakes in the text, and I think it could benefit from some professional copy editing, which is my background. Could the maintainers please provide feedback if they're willing to integrate these changes?

**Comment by greggman** [comment]
Thank you for the effort

I'm interested in typo fixes, grammar fixes, punctuation fixes.

I'm not interested in capitalizing three.js. In fact checking the three.js repo there are 87 instances of three.js in the docs and only 13 of Three.js, so no, it is not capitalized there. It's not capitalized in [the license](https://github.com/mrdoob/three.js/blob/dev/LICENSE) or even the [readme](https://github.com/mrdoob/three.js/blob/dev/README.md) nor even the [home page](https://threejs.org) nor in [the code itself](https://github.com/mrdoob/three.js/blob/dd8b5aa3b270c17096b90945cd2d6d1b13aaec53/docs/index.html#L475) which shows up in the JavaScript console on the [docs home page](https://threejs.org/docs)

I'm also not interested in being consistent with calling it `three.js` vs `three`. To me that's no different than saying "Timothy Smith was the President of the Smith Bread Company. Mr. Smith loved bread his whole life. Tim even loved waking up at 5am to start baking."  or "IBM Inc is a software consulting firm but IBM didn't start that way". Both would be tedious to stay consistent and use "Timothy Smith" in every instance and "IBM Inc" in every instance. It's not like the API is called THREEJS. You don't do `material = new THREEJS.MeshBasicMaterial()`. In fact the npm package is called 'three' and you `import * as THREE from 'three'` so to me `three` and `three.js` are like Tim and Timothy

Whether or not I'm interested in re-writes really depends on the rewrite. For example

> think of the word "frustum" as another 3D shape like a sphere, cube, prism, frustum.

vs

> think of the word "frustum" as another 3D shape like a sphere, cube, or prism

I find the that more confusing not less.  IMO the first makes it clear there's a set of things called shapes. Inside that set are (sphere, cube, prism, frustum). The second can be misinterpreted (to me at least) that frustum describes a sphere, a cube, and a prism

I'd consider this re-write

> think of the word "frustum" as another 3D shape as in sphere, cube, prism, frustum.

I would not consider this re-write

> think of the word "frustum" as another 3D shape as in sphere, cube, prism, and frustum.

The first is a list (a, b, c, d), the second is a sentence (a, b, c, and d). To me, the list is a clearer indication that "frustum" is a member of the set of things called shapes than the sentence.

I have similar issues with many of the other rewrites where many seem a matter of opinion vs an actual correction.

As for all the commas, while I agree some commas might be helpful the number of commas added seems excessive to me. The preceding sentence does not need a comma between "helpful" and "the". It can be written both ways. There is no rule that requires a comma, it is a choice to add one or not. 

TLDR;

1. three.js is not capitalized
2. A PR with only typo fixes and grammar fixes is appreciated and likely to be approved quickly
3. A PR with rewrites is more likely to face push back, arguing, matters of opinion
4. A PR with extra commas, case-by-case, the fewer the better.
5. smaller PRs are more likely to get approved (1 PR with 10 changes, 2 bad, will not be approved, 5 PRs with 2 changes each, 2 bad means 3 or 4 were approved)

**Comment by a-dpq** [comment]
> I'm not interested in capitalizing three.js. In fact checking the three.js repo there are 87 instances of three.js in the docs and only 13 of Three.js, so no, it is not capitalized there. It's not capitalized in the license or even the readme nor even the home page nor in the code itself which shows up in the JavaScript console on the docs home page

It would be more accurate to say it's inconsistent in the repo, not that it's not capitalized. And yes, it is capitalized on the homepage. I'm fine with either way, but it should be consistent throughout, and there is at least one instance of `Three.js` in the first chapter of this guide. I haven't checked the others yet.

> I'm also not interested in being consistent with calling it three.js vs three.

The problem with this approach is that, if you write "three" lowercase, it can be confused with the number three. Professional copy editors try to avoid this type of ambiguity, even when superficial, since it makes the text more readable.

The sentence with "frustum" is confusing as written, because it reads "another 3D shape" and then mentions a frustum, which is of course the same shape, not another shape. Your proposed rewrite still has this problem.

> As for all the commas, while I agree some commas might be helpful the number of commas added seems excessive to me. The preceding sentence does not need a comma between "helpful" and "the". It can be written both ways. There is no rule that requires a comma, it is a choice to add one or not.

This is not correct. Dependent clauses preceding main clauses need to be separated by a comma. See Chicago Manual of Style, 6.30. There are many such mistakes in the text, and I've tried as much as possible to restrict the addition of commas to required cases.

I'm always willing to discuss the rationale for changes, but in general I get the impression that my work here is not appreciated, so I won't contribute any more changes to this repository. I think it's too bad, since the material itself is valuable, but there are many changes that would be necessary to bring the text up to professional publishing standards.

**Comment by a-dpq** [comment]
Also please consider that many of the things that you're calling matters of opinion are in fact established standards in publishing. I'm happy to provide more references for these (such as the handling of numbers, for example) if you're open-minded enough to work with someone with professional competency in this field.

**Comment by a-dpq** [comment]
I find this behavior really disappointing, since I’m actually trying to be constructive and contribute to an important resource for the community. If you care about your project, you should care about making the text conform to established publishing standards, and you should recognize where your competencies can be augmented by someone with a different skill set. If you don't own and use English style guides because language is not your core competency, you may not be aware of or value these standards, but they are important.
